media/js/src/assetDetail/ViewItem.jsx

```diff
@@ -34,34 +34,75 @@ export default class ViewItem extends React.Component {
             userIsAuthor = authorId === window.MediaThread.current_user;
         }
 
+        const references = [];
         if (this.state.references) {
             this.state.references.forEach(function(reference, idx) {
                 references.push(
-                    <div className="card w-100 card-selection" key={idx}>
-                        <div className="card-body">
-                            <h5 className="card-title">
-                                <div>
-                                    {reference.title}
-                                </div>
-                            </h5>
-                            <h6>
-                                Assignment
-                            </h6>
-                            <p className="card-text">
-                                Collected 12/19/15 01:02 AM
-                            </p>
-                            <a href="#" className="btn btn-primary btn-sm">
-                                View
-                            </a>
-                        </div>
-                    </div>
+                    <li key={idx}>
+                        <a href={reference.url}>{reference.title}</a>
+                    </li>
                 );
             });
         }
 
+        let tags = '(None)';
+        let terms = '(None)';
+        if (this.props.asset && this.props.asset.annotations) {
+            const tagsArray = getTags(this.props.asset.annotations);
+            if (tagsArray.length > 0) {
+                tags = tagsArray.join(', ');
+            }
+
+            const termsArray = getTerms(this.props.asset.annotations);
+            if (termsArray.length > 0) {
+                terms = termsArray.join(', ');
+            }
+        }
+
+        let description = null;
+        if (this.props.asset && this.props.asset.metadata) {
+            let desc = find(this.props.asset.metadata, {key: 'Description'});
+            if (desc) {
+                desc = desc.value;
+            }
+            if (desc && desc.length) {
+                description = desc[0];
+            }
+        }
+
         return (
             <div className="tab-content" id="pills-tabContent">
-                <table className="table mt-1">
+                <h3>Tags</h3>
+                <p>
+                    {tags}
+                </p>
+
+                <hr />
+
+                <h3>Course Vocabulary (Terms)</h3>
+                <p>
+                    {terms}
+                </p>
+
+                <hr />
+
+                <h3>Course References</h3>
+                <Alert variant="warning" show={
```

Comment:
Tags & terms display plain text "(None)" when nothing exists. Can we make this consistent and pull the alert style here?

Reply:
I don't know, I think that alert is a bit much. Honestly I think we should just hide any headers that don't have any elements. But, here's what it would look like with the alerts:

<img width="542" alt="Screen Shot 2020-08-25 at 1 25 00 PM" src="https://user-images.githubusercontent.com/59292/91207287-85c6a780-e6d6-11ea-9d00-661d135a3c91.png">

Reply:
Oh maybe I misunderstood what you're saying here - should I make them all `(None)`?

Reply:
I'm not a big fan of the alerts either. I think make them all "None", or "There are no tags", "There are no terms" in plain text. I think hiding the sections if they are empty would be great.

Reply:
I've updated these to be consistent, in plain text.